core/commands/storage.go

```diff
@@ -561,6 +568,58 @@ func changeAddress(pinfo *peer.AddrInfo) error {
 	return nil
 }
 
+// for client to receive all the half signed contracts
+var storageUploadRecvContractCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "For client to receive collateral proof.",
```

Comment:
We don't use the term collateral proof anymore. It should be `For renter client to receive half signed contracts.`